test/over_react/component/shared_error_boundary_tests.dart

```diff
@@ -164,16 +184,20 @@ void sharedErrorBoundaryTests(BuilderOnlyUiFactory builder) {
           final component = jacket.getDartInstance();
           component.setState(component.newState()
             ..hasError = true
-            ..showFallbackUIOnError = true
           );
           expect(getByTestId(jacket.getInstance(), 'dummyChild'), isNull);
-          expect(getByTestId(jacket.getInstance(), 'fallbackNode'), isNotNull);
+          if (!isWrapper) {
+            // wrapper ErrorBoundary doesn't use `props.fallbackUIRenderer` because `ResolvableErrorBoundary`
+            // will display it on first error. Meaning wrapper ErrorBoundary will never be reached if it is set.
```

Comment:
The child error boundary catching and displaying the fallback renderer is tested here:
https://github.com/Workiva/over_react/pull/435/files#diff-89d97c9289557c366adc6964193a6057R372-R388